run_experiment.py

```diff
@@ -122,19 +131,39 @@ def parse_arguments():
     dir_opts.add_argument('-o', '--output_file_name', help='The full path name to use when saving the plot.')
 
     args = parser.parse_args()
+
+    if args.log_dir is None:
+        args.log_dir = args.model_dir + 'losses.log'
```

Comment:
again don't duplicate `'losses.log'` load teh variale `TRAIN_FILE` from `training.py` (I actually changed the name of `'losses.log'` to `'training_losses.log'` as we also have the test one, but I was not going to change the variable name )